doc/interfaces/gibbs_unringing_flow.rst

```diff
@@ -0,0 +1,53 @@
+.. _gibbs_unringing_flow:
+
+===============
+Gibbs Unringing
+===============
+
+This tutorial walks through the steps to perform Gibbs unringing using DIPY.
+
+You can try this using your own data; we will be using the data in DIPY.
+You can check how to :ref:`fetch the DIPY data<data_fetch>`.
+
+---------------------------
+Suppress Gibbs Oscillations
+---------------------------
+
+Magnetic Resonance (MR) images are reconstructed from the Fourier coefficients
+of acquired k-space images. Since only a finite number of Fourier coefficients
+can be acquired in practice, reconstructed MR images can be corrupted by Gibbs
+artefacts, which is manifested by intensity oscillations adjacent to edges of
+different tissues types [1]_. Although this artefact affects MR images in
+general, in the context of diffusion-weighted imaging, Gibbs oscillations
+can be magnified in derived diffusion-based estimates [1]_, [2]_.
+
+The workflow for the Gibbs unringing requires path to the input volumes. This
+path may contain wildcards to process multiple inputs at once.
+
+You may want to create an output directory to save the resulting denoised 
+volume::
+
+    mkdir gibbs_ringing_output
+
+To run the Gibbs unringing method, we need to specify path to the input volumes.
+This path may contain wildcards to process multiple inputs at once. You can also
+specify the optional arguments. In this case, we will be specifying the
+``num_threads`` and ``out_dir``.
+
+    dipy_gibbs_ringing data//tissue_data/t1_brain_denoised.nii.gz --num_threads 2 --out_dir "gibbs_ringing_output"
```

Comment:
dipy_gibbs_ringing data/tissue_data/t1_brain_denoised.nii.gz --num_threads 2 --out_dir "gibbs_ringing_output"

(One forward slash in excess).

Reply:
Removed extra slash in b2f541e